compose/compiler/design/target-inference.md

```diff
@@ -203,4 +203,4 @@ by using a circular-linked-list to track the binding variables (two circular-lin
 be concatenated together in `O(1)` time by swapping next pointer of just one element in each).
 The bindings can be reversed (e.g. swapping the next pointers back), but the code is not
 included to do so, so if backtracking is later required, `Bindings` would need to be updated
-accordingly. Details are provided in the class.
\ No newline at end of file
+accordingly. Details are provided in the class.
```

Comment:
This change was applied automatically in the web editor on github. Can add it back manually if you want

Reply:
No worries. Change is innocent enough :)